draft-ietf-rats-corim.md

```diff
@@ -1843,15 +1828,37 @@ The `cm` field is set based on the type of Conceptual Message inputted or to be
 
 All of the extracted and validated tags are loaded into an *appraisal context*.
 The Appraisal Context contains an internal representation of the inputted Conceptual Messages.
-The selected tags are mapped to the internal representation, making them suitable for appraisal processing.
+The selected tags are mapped to an internal representation, making them suitable for appraisal processing.
 
 [^issue] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/96
 
-#### Reference and Endorsed Values Tranformation
+#### Reference Triples Tranformation {#sec-ref-trans}
+
+An available `rv` list entry {{sec-ir-ref-val}} is allocated.
```

Comment:
This (and similar instances below) will render as:
```
An available rv list entry Section 8.2.1.2 is allocated.
```
which is not great.

Please wrap all the references to sections in parentheses so that it reads like:
```
An available rv list entry (Section 8.2.1.2) is allocated.
```